Core/Debugger/WebSocket/InputSubscriber.cpp

```diff
@@ -192,6 +192,7 @@ void WebSocketInputState::ButtonsPress(DebuggerRequest &req) {
 	}
 	press.button = info->second;
 
+	// TODO: Route into the control mapper's PSPKey function instead.
```

Comment:
I certainly wouldn't want this to have any dependency on `UI` or direct connection to EmuScreen?  What, would there be a global instance of ControlMapper then in Core?

-[Unknown]

Reply:
Yeah I've changed my mind on this already. I'll get rid of these comments.